PromisesObjC.podspec

```diff
@@ -23,9 +23,14 @@ Pod::Spec.new do |s|
   s.public_header_files = "Sources/#{s.module_name}/include/**/*.h"
   s.private_header_files = "Sources/#{s.module_name}/include/FBLPromisePrivate.h"
   s.source_files = "Sources/#{s.module_name}/**/*.{h,m}"
+  s.module_map = "Sources/#{s.module_name}/include/framework.modulemap"
+  s.preserve_path = "Sources/#{s.module_name}/include/framework.modulemap"
```

Comment:
Is this needed?

Reply:
Hi @temrich! do you specifically mean `preserve_path` or do you mean both `preserve_path` and `module_map`?

Ill try playing with a combination of both.

Reply:
`preserve_path`, but also curious why `module_map` is needed as well. Also, which version of CocoaPods are you using?